### ISSUE TITLE
Error: service type not selected when using with pushbullet

### ISSUE THREAD

**Issue by tnguyen14** [jreese/znc-push#198]
I loaded this module and set the service type to be `pushbullet`, with `target` and `secret` set. However, whenever I get a highlight mention, I see the error `Error: service type not selected`.

When I try to send a notification manually with the `send` command, the response is `Ok`, but the notification does not come through.

Note: I run znc as a docker container. I am not sure if that would have any impact in terms of networking required to make znc-push work or not.

**Comment by tnguyen14** [comment]
Trying to see if this could be due to the fact that I first loaded the module at the network level, then went on the load at the user level.

I've unloaded it from the network level, and these are the settings as seen from ZNC webadmin. It still doesn't seem to be working yet.

From "Your Settings" page:
<img width="696" alt="screen shot 2016-03-15 at 12 48 57 pm" src="https://cloud.githubusercontent.com/assets/1652595/13786093/88c768fa-eaac-11e5-854b-064604563c47.png">

From the Network page:
<img width="707" alt="screen shot 2016-03-15 at 12 50 59 pm" src="https://cloud.githubusercontent.com/assets/1652595/13786111/9bdd68ea-eaac-11e5-9db8-9939f828fd65.png">
